tools/experiments/experiments-config.js

```diff
@@ -242,11 +242,6 @@ export const EXPERIMENTS = [
     name: 'Opts-in users into using AnimationWorklet',
     cleanupIssue: 'X',
   },
-  {
-    id: 'amp-consent-v2',
-    name: 'Enables CMP support to amp-consent component',
-    spec: 'https://github.com/ampproject/amphtml/issues/17742',
-  },
```

Comment:
This is fine?

Reply:
Yes, we remove the config from this file, so that the experiment.html file won't pick the experiment flag.